components/json/rs/src/lib.rs

```diff
@@ -0,0 +1,121 @@
+#[cxx::bridge(namespace = json)]
+mod ffi {
+    extern "Rust" {
+        fn convert_uint64_value_to_string(path: &str, json: &str) -> String;
+        fn convert_int64_value_to_string(path: &str, json: &str) -> String;
+        fn convert_string_value_to_uint64(path: &str, json: &str) -> String;
+        fn convert_string_value_to_int64(path: &str, json: &str) -> String;
+    }
+}
+
+// Parses and re-serializes json with the value at path converted from a uint64
+// to a string representation of the same number.
+// Returns an empty String if such conversion is not possible.
+// A path is a Unicode string with the reference tokens separated by /.
+// Inside tokens / is replaced by ~1 and ~ is replaced by ~0.
+// Examples: convert_uint64_value_to_string("/a/b", json) for { a : { b : 1 }}
+//           convert_uint64_value_to_string("/a/0", json) for { a : [ 1 ]}
+pub fn convert_uint64_value_to_string(path: &str, json: &str) -> String {
+    let result_value = serde_json::from_str(&json);
+    if let Err(result_value) = result_value {
+        return String::new();
+    }
```

Comment:
Without the logging, this warns about the unused inner result_value. `if result_value.is_err() { ... }` would  be better.

Reply:
done